src/main/resources/checkstyle/eg-checkstyle.xml

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC "-//Puppy Crawl//DTD Check Configuration 1.2//EN"
+  "http://www.puppycrawl.com/dtds/configuration_1_2.dtd">
+
+  <!--
+    Checkstyle configuration for Hadoop that is based on the sun_checks.xml file that is bundled with Checkstyle and
```

Comment:
checkstyle configuration for `Hadoop`??

Reply:
heh, good catch, that's been in there for years! Will reword.